diff --git b/bot-rs/src/schema.rs a/bot-rs/src/schema.rs
index 855b7f8..f6f591b 100644
--- b/bot-rs/src/schema.rs
+++ a/bot-rs/src/schema.rs
@@ -81,6 +81,47 @@ table! {
     }
 }
 
+table! {
+    deck_records (id) {
+        id -> Int4,
+        match_wins -> Int8,
+        match_losses -> Int8,
+        game_wins -> Int8,
+        game_losses -> Int8,
+    }
+}
+
+table! {
+    leaderboard (id) {
+        id -> Int4,
+        name -> Text,
+        league -> Int4,
+        wins -> Int8,
+        complete_runs -> Int8,
+    }
+}
+
+table! {
+    cards (id) {
+        id -> Int8,
+        name -> Text,
+        number -> Text,
+        setcode -> Text,
+        isarena -> Int8,
+        scryfallid -> Uuid,
+        scryfalloracleid -> Uuid,
+        manacost -> Nullable<Text>,
+        types -> Text,
+        convertedmanacost -> Float8,
+        side -> Nullable<Text>,
+        uuid -> Uuid,
+        layout -> Nullable<Text>,
+    }
+}
+
+joinable!(deck_records -> decks (id));
+joinable!(leaderboard -> users (id));
+joinable!(leaderboard -> leagues (league));
 joinable!(deck_contents -> decks (deck));
 joinable!(deck_view_tokens -> decks (deck));
 joinable!(decks -> leagues (league));
@@ -97,4 +138,7 @@ allow_tables_to_appear_in_same_query!(
     leagues,
     matches,
     users,
+    cards,
+    deck_records,
+    leaderboard,
 );
